### ISSUE TITLE
why not publish the code same time as paper. why even open a github repo if have no code??

### ISSUE THREAD

**Issue by kyegomez** [NousResearch/DisTrO#2]

**Comment by peterwilli** [comment]
In the paper they stated they are not sure yet how this works. Also, the paper is not the real paper but a preliminary paper. I can see why they dont want to bother the community with unproven and unknown things.

You're more than welcome to reach out to them and help out!

**Comment by kyegomez** [comment]
@peterwilli i would love to help them out, but they have casted me out of their discord and refuse to work with me because i implement papers? I've tried to speak with teknium and offer my support but they are not open-minded to collaboration.

**Comment by feffy380** [comment]
A warning to anyone reading: kyegomez is a grifter who quickly shits out a fake SEO optimized repo whenever a high-profile paper gets published, then begs for funding and silently closes bug reports calling out the fact that none of his hundreds of repos contain functioning code. If you look at the repos, they're all barely edited templates with some paper excerpts and AI-generated descriptions in an attempt to look legitimate.

**Comment by kyegomez** [comment]
> A warning to anyone reading: kyegomez is a grifter who quickly shits out a fake SEO optimized repo whenever a high-profile paper gets published, then begs for funding and silently closes bug reports calling out the fact that none of his hundreds of repos contain functioning code (the fact that he uploaded >300 repos within the last 3 years is already a huge red flag). If you look at the repos, they're all barely edited templates with some paper excerpts and AI-generated descriptions in an attempt to look legitimate. If they have bug reports, they're mainly reports that the code doesn't work and get closed without resolution.
> 
> 
> 
> Here's an example of a researcher having to [threaten legal action](https://github.com/kyegomez/tree-of-thoughts/issues/56#issuecomment-1570647622) to get proper attribution when he essentially plagiarized their repo.
> 
> 
> 
> TLDR: OP is a well-known grifter and is upset that the repo doesn't contain code for him to plagiarize

Can you provide atleast one error in any of my repos, just one please? I see a lot of claims but no evidence

**Comment by arilotter** [comment]
I'm closing the discussion here because it's moved far off-topic. We're working on the paper as fast as we can, and will publish the paper & code as soon as it's ready, hopefully very soon.

**Comment by arilotter** [comment]
DeMo, the foundational work for DisTrO, is now available with a paper & codebase. See the README.md file for more info!
